CMSgov/easi-app
# [EASI-1684] - Add Email Recipients when taking an action

# EASI-1684

## Changes and Description

- Functionality to add additional contacts to email when taking an action
- This ticket includes only the functionality to add the contacts - displaying/selecting contacts is handled in [EASI-2019](https://jiraent.cms.gov/browse/EASI-2019)
- Contacts section currently displays contacts because component is shared with system intake. I left it so that we can see the added contacts during testing, but this will be modified to match the [Figma](https://www.figma.com/file/kNsCJCCRvGJCWwPQO6Gqfr/GRT-Admin-Experience?node-id=58%3A8717) in EASI-2019.



## How to test this change

- Sign in as GRT admin
- Click on open intake request
- Click "Actions" link in sidebar
- Select action type and continue
- Click "Add another recipient" button and complete form
- Submitting form should add contact to system intake

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
